integration_tests/src/main/python/benchmark.py

```diff
@@ -121,6 +123,9 @@ def main():
             else:
                 cmd.append("--iterations {}".format(args.iterations))
 
+            if args.gc_between_runs is True:
```

Comment:
I would move this up so it happens before the `--iterations` flag is added since that is currently the last flag and is not followed by whitespace, so we could end up with `--iterations=1--gb-between` with this code.